less/pages/safety/safety.less

```diff
@@ -105,6 +105,26 @@
       margin: 0 0.2rem 0 0;
       height: 1rem;
     }
+    input[type=checkbox] {
+      display: none;
+    }
+    label:before {
+      content: "";
+      display: inline-block;
+      width: 24px;
+      height: 24px;
+      position: relative;
+      background-color: #B94FBB;
+      border: 1px solid white;
+      margin-right: 7px;
```

Comment:
Could make this `margin-right: 0.3rem;` or `margin-right: 0.32rem;` if you want to get precise.